templates/repo/issue/view_content.tmpl

```diff
@@ -153,6 +158,11 @@
 						</a>
 						<div class="content">
 							<form class="ui segment form" id="comment-form" action="{{$.RepoLink}}/issues/{{.Issue.Index}}/comments" method="post">
+								{{if .Issue.IsPrivate}}
+									<div class="ui message orange">
```

Comment:
Don't use colors, use intentions, e.g. `warning`. It's very likely that dark theme changes are necessary as well, I can check that later.

Reply:
Oh, for some reason `.warning` has a `display: none`:
https://github.com/go-gitea/gitea/blob/6c2196146212c47304af12f5533cd46e5ff02807/web_src/fomantic/build/semantic.css#L10674-L10677

I've probably did something wrong on my end to have that `display: none` removed.

Reply:
Hmm, that seems like a very strange style to come from fomantic. I guess that rule must target form validation and there might be some other rule that overrides it. I guess you can keep it at yellow in this case.